torchrl/data/rlhf/utils.py

```diff
@@ -0,0 +1,293 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+from typing import Tuple
+
+import torch
+
+from tensordict import TensorDict
+from torch import Tensor
+from torch.nn import functional as F
+
+from torchrl.data.rlhf.prompt import PromptData
+from transformers import GenerationConfig
+
+
+class RolloutFromModel:
+    """A class for performing rollouts with language models.
+
+    Args:
+        model (transformers.Transformer): the model to be used. Should have a
```

Comment:
shouldn't we state what defines a "language model"? E.g., 

```suggestion
    """A class for performing rollouts with causal language models, i.e., a model that takes in input tokenized text
        and whose task is to predicting the next word in a sentence having read the n previous words.

    Args:
        model (transformers.Transformer): the model to be used. Should have a
```